### ISSUE TITLE
[pidgin] too many group tabs

### ISSUE THREAD

**Issue by dchmelik** [EionRobb/purple-discord#415]
Purple-discord is nice in BitlBee because you only add the rooms you want.
&nbsp;&nbsp;&nbsp; It's not nice in Pidgin, because despite standard for servers to get one group tab--and discord.com is one server regardless how many places you joined--I get too many group tabs: dozens which should be option to all be in one.  I'm in several discord places which would be bad enough as separate group tabs but most have several sections themselves the become entire group tabs (but shouldn't have to) so end up dozens too many group tabs.
&nbsp;&nbsp;&nbsp; Lately it's a critical problem, because pidgin seems to load tabs in order connected, which is useless, so I have to rearrange them, but with the latest stable pidgin, seems the more tabs you have to move, not only does it usually take many tries but usually moving any group tab eventually halts/crashes... not just moving discord tabs but for any/all protocols (but the more there are the longer it takes with almost certain chance of halt/crash.)

**Comment by axet** [comment]
Yep. We already have Room list dialog. All rooms showed there. Pidgin chat should only have rooms I post messages, not all of them.

**Comment by skipster1337** [comment]
The fact that it opens so many tabs also leads to you getting temporarily IP banned because it's making way too many requests to Discord. I'm using an account with a decent amount of servers and when I log in, all the channels open up at once, crashing Pidgin and getting me *inevitably* IP banned.
